### PR TITLE
Bump API to v1.26

### PR DESCRIPTION
**Pull request by lowenna**
Signed-off-by: John Howard <jhoward@microsoft.com>

Think I got everything. I wasn't sure about the change to `runconfig\opts\parse.go`, so if someone could verify, would be greatly appreciated!

@tiborvass @vieux @thaJeztah

### CODE REVIEW

**Comment by tiborvass** [comment]
Thanks @jhowardmsft. The SetAnnotations should stay at 1.25, it's the version starting from which the feature exists.

**Comment by jhowardmsft** [comment]
@tiborvass Got it, thanks! Updated to revert those `SetAnnotations` updates.

File: cli/command/checkpoint/cmd.go [hunk 1]
```diff
@@ -16,7 +16,7 @@ func NewCheckpointCommand(dockerCli *command.DockerCli) *cobra.Command {
 			cmd.SetOutput(dockerCli.Err())
 			cmd.HelpFunc()(cmd, args)
 		},
-		Tags: map[string]string{"experimental": "", "version": "1.25"},
+		Tags: map[string]string{"experimental": "", "version": "1.26"},
```
Comment (thaJeztah) [inline cli/command/checkpoint/cmd.go]: I think these determine the _minimum_ version that supports these commands, so should not be changed (same for the others below

Comment (jhowardmsft) [inline cli/command/checkpoint/cmd.go]: @thaJeztah Updated. I was clearly way too over-zealous! What about the `NewVersionError` calls?

**Review by thaJeztah — CHANGES_REQUESTED** [review]
hmf, looks like my comments were still "pending" on GitHub

**Comment by jhowardmsft** [comment]
@thaJeztah Yup, all your comments addressed in latest update.

**Comment by tiborvass** [comment]
LGTM

**Review by thaJeztah — APPROVED** [review]
LGTM

**Pull request merged**